opennms-web-api/src/main/java/org/opennms/web/enlinkd/EnLinkdElementFactory.java

```diff
@@ -718,7 +721,8 @@ private BridgeLinkNode addBridgeRemotesNodes(
         for (String sharedmac: macsToIpNetTOMediaMap.keySet()) {
             final BridgeLinkRemoteNode remlinknode = new BridgeLinkRemoteNode();
             if (macsToIpNetTOMediaMap.get(sharedmac).isEmpty()) {
-                OnmsSnmpInterface snmp = getFromPhysAddress(sharedmac);
+                var l = physAddrToSnmpInterfaces.get(sharedmac);
+                OnmsSnmpInterface snmp = l != null && l.size() == 1 ? l.get(0) : null;
```

Comment:
Same as in the caches. This can be done while building the map. In this case we don't even need the keys with a null value but can skip the whole entry instead.